src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -7077,6 +7077,16 @@ public object VisitParenExpression(ParenExpressionAst parenExpressionAst)
         {
```

Comment:
@rjmholt, your last commit had 5 failures in `PowerShell-CI-windows`
Experimental Feature: && and || operators - Feature-Enabled.Gets the correct output and errorID with terminating statement 'testexe -returncode 0 && Test-TerminatingBadCommand'
```powershell
Expected '0', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1: line 161
161:         $result | Should -Be $Output
```
Experimental Feature: && and || operators - Feature-Enabled.Gets the correct output and errorID with terminating statement 'testexe -returncode 0 && throw 'Bad''
```powershell
Expected '0', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1: line 161
161:         $result | Should -Be $Output
```
Experimental Feature: && and || operators - Feature-Enabled.Runs the statement chain 'testexe -returncode 0 && testexe -returncode 1 &' as a job
```powershell
Expected @('0', '1'), but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1: line 184
184:         $resultJob | Wait-Job | Receive-Job | Should -Be $Output
```
Experimental Feature: && and || operators - Feature-Enabled.Runs the statement chain 'testexe -returncode 1 && testexe -returncode 0 &' as a job
```powershell
Expected '1', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1: line 184
184:         $resultJob | Wait-Job | Receive-Job | Should -Be $Output
```
Experimental Feature: && and || operators - Feature-Enabled.Runs the statement chain '$x = testexe -returncode 0 && Write-Output "mice" &' as a job
```powershell
Expected @('0', 'mice'), but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1: line 184
184:         $resultJob | Wait-Job | Receive-Job | Should -Be $Output
```